contract/contracts/@eip2981/IERC2981Royalties.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GNU General Public License v3.0
+pragma solidity ^0.8.10;
+
+/// @title IERC2981Royalties
+/// @dev Interface for the ERC2981 - Token Royalty standard
+interface IERC2981Royalties {
+    /// @notice Called with the sale price to determine how much royalty
+    //          is owed and to whom.
```

Comment:
Not sure if it works to use `//` for this line. I personally would just write everything on one line or use a comment with less words. I could imagine that the EVM can't interpret this line as part of the natspec comment because it is just a single line command

Reply:
Yeah good point, though sorry this wasn't my code and was just straight up copied from the repo that has the ERC2981 implementation.

But will change that.